qe_metrics/cli.py

```diff
@@ -35,15 +36,20 @@
 def main(products_file: str, config_file: str, pdb: bool, verbose_db: bool) -> None:
     """Gather QE Metrics"""
 
-    # Adding noqa: F841 to ignore the unused variable until next PR, otherwise pre-commit will fail
-    with Database(config_file=config_file, verbose=verbose_db) as database:
-        jira = Jira(config_file=config_file)  # noqa: F841
-        products = database.Products.from_file(products_file=products_file)
-        for product in products:
+    with Database(config_file=config_file, verbose=verbose_db) as database, Jira(
+        config_file=config_file
+    ) as jira, orm.db_session:
+        # TODO: Run a cleanup of the database to remove old entries
+
+        for product in database.Products.from_file(products_file=products_file):
             for severity, query in product.queries.items():
-                # TODO: Execute Jira query and populate the database
-                pass  # TODO: Remove this line once the code is implemented
-    # TODO: Run a cleanup of the database to remove old entries
+                _logger.info(f'Executing Jira query for "{product.name}" with severity "{severity}"')
+                database.JiraIssues.create_update_issues(
```

Comment:
I guess `create_update_issues` can raise exception, we need to catch it and continue to the next product. (and have a proper log that some product failed)